### DISCUSSION TITLE
500: Internal Error

### DISCUSSION THREAD

**Discussion by aazam476** [open-webui/open-webui#4772] [Q&A]
After logging in, I get to a page that just says "500: Internal Error"
All HTTP logs report 200/304
<img width="1799" alt="Screenshot 2024-08-21 at 11 14 30 AM" src="https://github.com/user-attachments/assets/ade0c946-7e52-4784-b937-71e874a187ef">

**Comment by justinh-rahb** [comment]
Are you setting any special environment variables for this install?

Can you try clearing your cache/site data and logging in again?

**Comment by magnus919** [comment]
I get this, too. But if I force quit safari and restart it, sometimes it works again. This is so weird.
